modules/custom/openy_activity_finder/src/Plugin/ActivityFinderBackend/OpenyActivityFinderSolrBackend.php

```diff
@@ -1,36 +1,42 @@
 <?php
 
-namespace Drupal\openy_activity_finder;
+namespace Drupal\openy_activity_finder\Plugin\ActivityFinderBackend;
 
 use DateTimeInterface;
 use Drupal\Component\Datetime\TimeInterface;
-use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Annotation\Translation;
 use Drupal\Core\Datetime\DateFormatterInterface;
 use Drupal\Core\Cache\CacheBackendInterface;
 use Drupal\Core\Database\Connection;
-use Drupal\Core\Entity\EntityTypeManager;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
 use Drupal\Core\Entity\Query\QueryFactory;
 use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\openy_activity_finder\ActivityFinderBackendPluginBase;
+use Drupal\openy_activity_finder\Annotation\ActivityFinderBackend;
 use Drupal\search_api\Entity\Index;
 use Drupal\Core\Url;
 use Drupal\Core\Datetime\DrupalDateTime;
-use Drupal\Core\Logger\LoggerChannelInterface;
 use Drupal\search_api\SearchApiException;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
 
 /**
- * {@inheritdoc}
+ * Activity Finder Solr backend plugin.
+ *
+ * @ActivityFinderBackend(
+ *   id = "openy_activity_finder.solr_backend",
```

Comment:
Let's use shorter name here `id = "solr"`. Its already clear that this is backend plugin and it's related to the Activity Finder. For Daxko we can use just `daxko` as well.
But we should provide upgrade path for activity configs, let's implement a hook_update and replace it in the configuration:
- `openy_activity_finder.solr_backend` with `solr`
- `openy_daxko2.openy_activity_finder_backend` with `daxko`